### PR TITLE
Better device status reporting at local WS connect

### LINKED ISSUE #58: [Bug] Show example/pre-set RMT pin definitions on Setup page
### Pre-submission checklist

- [X] I checked that there is no other Feature Request describing my wish.
- [X] I checked that this feature is, in fact, not supported.
- [X] I accept that this issue may be closed if any of the above are found to be untrue.

### What should be possible that currently is not?

If using a PCB/pre-existing design, there should be a placeholder pin set in the RMT pin input on the Setup page.

If not, it should fall back to a known good pin for the board.

### Other remarks

Maybe in Wiki, we can have a listing of recommended RMT pins? Mark out which pins are recommended? (Not used by SPI flash, aren't input-only, etc)

### PR DESCRIPTION
**Pull request by hhvrc**
fixes #58 
Reports connected pin when captive portal is launched

### CODE REVIEW

File: WebUI/src/routes/+page.svelte [hunk 1]
```diff
@@ -36,12 +44,17 @@
 <div class="flex flex-col items-center justify-center h-full">
   <div class="flex-col space-y-5 w-full max-w-md">
     <WiFiList />
+
     <div class="flex flex-col space-y-2">
-      <h3 class="h3">Other settings</h3>
+      <h3 class="h3">Gateway Pairing</h3>
```
Comment (LucHeart) [inline WebUI/src/routes/+page.svelte:49]: "gateway pairing"? u mean the regular pairing?

File: include/_fbs/DeviceToLocalMessage_generated.h [hunk 2]
```diff
@@ -497,6 +455,87 @@ inline ::flatbuffers::Offset<WifiNetwork> CreateWifiNetworkDirect(
...
+    ::flatbuffers::FlatBufferBuilder &_fbb,
+    bool poggies = false,
+    ::flatbuffers::Offset<OpenShock::Serialization::Local::WifiNetwork> connected_wifi = 0,
+    bool gateway_paired = false,
+    uint8_t rftx_pin = 0) {
+  ReadyMessageBuilder builder_(_fbb);
+  builder_.add_connected_wifi(connected_wifi);
+  builder_.add_rftx_pin(rftx_pin);
+  builder_.add_gateway_paired(gateway_paired);
+  builder_.add_poggies(poggies);
```
Comment (LucHeart) [inline include/_fbs/DeviceToLocalMessage_generated.h:530]: whats poggies nowadays?

**Review by LucHeart — CHANGES_REQUESTED** [review]
Im not sure if gateway pairing is an appropriate name for that label

File: WebUI/src/routes/+page.svelte (hunk 1 above)
Comment (hhvrc) [reply WebUI/src/routes/+page.svelte:49]: Reverting this.

File: include/_fbs/DeviceToLocalMessage_generated.h (hunk 2 above)
Comment (hhvrc) [reply include/_fbs/DeviceToLocalMessage_generated.h:530]: its a boolean flag i decided to leave in the message just as a "everythings working!" kinda thing, discord has similar with the much more understandable "hello" message

**Review by LucHeart — APPROVED** [review]

**Pull request merged**